apps/categories2/templates/index.jade

```diff
@@ -7,6 +7,27 @@ append locals
   - assetPackage = 'categories2'
 
 block body
-  .categories-page.main-layout-container
-    h1.categories-page__header
-      | The Art Genome Project
+  .l-categories-page.main-layout-container
+
+    .l-categories-page__nav
```

Comment:
Ditto here, not a big deal but ideally we namespace by `categories2-l-page__nav`—again, just ensuring that styles never unintentionally conflict.